cli/script/command-executor.ts

```diff
@@ -178,6 +178,24 @@ function appList(command: cli.IAppListCommand): Promise<void> {
         });
 }
 
+function checkIfAppExists(appName: string): Promise<App> {
+    log(chalk.cyan(`Checking if app with the name "${appName}" exists...\n`));
+    return Promise<App>((resolve, reject) => {
+        sdk.getApps()
```

Comment:
Actually thinking about it more, it might make even more sense to call [`sdk.getDeployment(<appName>, <deploymentName>)`](https://github.com/Microsoft/code-push/blob/master/sdk/script/management-sdk.ts#L273), with a comment explaining why, because this way the server will validate both the app and deployment and return the appropriate error message.